### PR TITLE
Settings: add pairing and unpairing feature back

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

File: ts/components/session/RegistrationTabs.tsx [hunk 1]
```diff
@@ -826,16 +833,26 @@ export class RegistrationTabs extends React.Component<{}, State> {
       await this.accountManager.requestPairing(primaryPubKey);
       const pubkey = window.textsecure.storage.user.getNumber();
       const words = window.mnemonic.pubkey_to_secret_words(pubkey);
-      window.console.log('pubkey_to_secret_words');
       window.console.log(`Here is your secret:\n${words}`);
+      window.pushToast({
+        title: `Here is your secret: "${words}"`,
```
Comment (vincentbavitz) [inline ts/components/session/RegistrationTabs.tsx]: Here, use ``window.i18n()``:

```
title: window.i18n(`${secretPrompt} ${words}`)
```

File: ts/components/session/RegistrationTabs.tsx [hunk 2]
```diff
@@ -826,16 +833,26 @@ export class RegistrationTabs extends React.Component<{}, State> {
...
+        id: 'yourSecret',
+        shouldFade: false,
+      });
     } catch (e) {
       window.console.log(e);
       //onError(e);
```
Comment (vincentbavitz) [inline ts/components/session/RegistrationTabs.tsx]: We can remove onError(e);

File: ts/components/session/SessionRegistrationView.tsx [hunk 3]
```diff
@@ -6,6 +6,7 @@ import { SessionIconButton, SessionIconSize, SessionIconType } from './icon';
 
 export const SessionRegistrationView: React.FC = () => (
   <div className="session-content">
+    <div id="session-toast-container" />
```
Comment (vincentbavitz) [inline ts/components/session/SessionRegistrationView.tsx]: Purpose of moving the toast container into here?

Comment (Bilb) [reply ts/components/session/SessionRegistrationView.tsx]: We didn't have it in the session registration view. So, toasts made before the registration is complete are not shown. And I need to show a toast during registration.

Comment (vincentbavitz) [reply ts/components/session/SessionRegistrationView.tsx]: Oh, great -- sounds good.

**Pull request merged**